MapboxCoreNavigationTests/TileRoutingVersionTests.swift

```diff
@@ -0,0 +1,56 @@
+@testable import MapboxCoreNavigation
+import MapboxDirections
+import XCTest
+
+class TileRoutingVersionTests: XCTestCase {
+    func testCurrentVersionReturnsExpectedValueFirstTime () {
+        // Given
+        let routeTilesVersion = RouteTilesVersion(with: DirectionsCredentials())
+        let expectedCurrentVersion = "2020_07_03-03_00_00"
+        
+        // When
+        let currentVersion = routeTilesVersion.currentVersion
+        
+        // Then
+        XCTAssertEqual(currentVersion, expectedCurrentVersion)
+    }
+    
+    func testCurrentVersionStoresValuePersistently () {
+//        // Given
+//        let routeTilesVersion = RouteTilesVersion(with: DirectionsCredentials())
+//        let expectedCurrentVersion = "2020_07_03-03_00_00"
+//
+//        // When
+//        let currentVersion = routeTilesVersion.currentVersion
+//
+//        // Then
+//        XCTAssertEqual(currentVersion, expectedCurrentVersion)
+    }
+    
+    func testCurrentVersionReadsValuePersistently () {
+//        // Given
+//        let routeTilesVersion = RouteTilesVersion(with: DirectionsCredentials())
+//        let expectedCurrentVersion = "2020_07_03-03_00_00"
+//
+//        // When
+//        let currentVersion = routeTilesVersion.currentVersion
+//
+//        // Then
+//        XCTAssertEqual(currentVersion, expectedCurrentVersion)
+    }
+
+    func testGetAvailableVersionsReturnsArrayOfAvailableVersions () {
+        // Given
+        let expectation = XCTestExpectation(description: "")
+        let routeTilesVersion = RouteTilesVersion(with: DirectionsCredentials())
+        
+        // When
+        routeTilesVersion.getAvailableVersions { availableVersions in
+            print(availableVersions)
+            expectation.fulfill()
```

Comment:
There are no actual checks performed except `expectation` usage. Should we verify available versions?

Reply:
I think these tests were a work in progress as part of #2533, but they got folded into this PR at some point. Like the PassiveLocationManagerTests, this class depends on network access, but fortunately it’s straightforward to mock the response in this case.